### DISCUSSION TITLE
More documentation about deployment

### DISCUSSION THREAD

**Discussion by alikalik9** [zauberzeug/nicegui#758] [Ideas / Feature Requests]
As someone who has never worked with Docker, SSL certificates, Traefikx or NGINX I find it really difficult to deploy my NiceGUI application based on the existing documentation. 
The structure of my Python directory also includes additional .py files that contain various classes, in addition to my main.py. These classes contain Nicegui components that I import into main.py. I'm not sure if I can even deploy this structure.

It would be very appreciated, if you could document how to deploy a NiceGUI app with more detail. In heroku you can deploy directly from a Github Repository, but it didn't work for me. 
An example (with Heroku, or any other Provider or technique) would be very usefull. 
Thanks!

**Comment by rodja** [comment]
Yes. More documentation is always welcome. We try our best. And to simplify deployment is also a good goal. We are currently working on something in this direction. But we will keep the details hidden for now to make it a surprise 😉

**Reply by alikalik9** [reply]
Sounds promising 🙂
